src/mixins/filing-template-mixin.ts

```diff
@@ -1066,13 +1072,23 @@ export default class FilingTemplateMixin extends DateMixin {
     // filter out removed classes and delete "action" property
     const shareClasses = this.getShareClasses
       .filter(x => x.action?.toUpperCase() !== ActionTypes.REMOVED)
-      .map((x) => { const { action, ...rest } = x; return rest })
+      .map((x) => {
+        const { action, ...rest } = x
+        rest.hasMaximumShares = !!rest.hasMaximumShares // change null -> false
+        rest.hasRightsOrRestrictions = !!rest.hasRightsOrRestrictions // change null -> false
```

Comment:
These 2 lines (and below for series) are what caused the schema error. These values must be boolean but in some cases they were null.